### ISSUE TITLE
Inazuma Eleven 3: Team Ogre Attack - EUR VERSION (LAG & Stuttering)

### ISSUE THREAD

**Issue by joyful-cambion** [weihuoya/citra#326]
Hello, 
First of all thanks for you magnificent job on this project, I have tried IE3 on Citra MMJ latest build (10/06/2021) And got horrible framerate/stuttering and fps of (10-12) this was also mentioned on the official citra repository, I think it's an emulator issue, Anyway thought of letting you know...
--

**Comment by TheOG00** [comment]
@joyful-cambion 
Phone model please so i can help

**Comment by CyberBlueCodes** [comment]
Do any other game work better than that on your device?

**Comment by joyful-cambion** [comment]
@CyberBlueCodes Yes I am also playing pokémon omega ruby and it's playable at 30-32 fps
